ppr-api/src/ppr_api/models/financing_statement.py

```diff
@@ -52,42 +58,46 @@ class FinancingTypes(Enum):
     __versioned__ = {}
     __tablename__ = 'financing_statement'
 
-    financing_id = db.Column('financing_id', db.Integer, primary_key=True, server_default=db.FetchedValue())
-    financing_type_cd = db.Column('financing_type_cd', db.String(3), nullable=False) #, db.ForeignKey('financing_type.financing_type_cd'))
-    state_type_cd = db.Column('state_type_cd', db.String(3), nullable=False) #, db.ForeignKey('state_type.state_type_cd'))
-    lien_amount = db.Column('lien_amount', db.Numeric, nullable=True)
-    surrender_dt = db.Column('surrender_dt', db.Date, nullable=True)
+#    financing_id = db.Column('financing_id', db.Integer, primary_key=True, server_default=db.FetchedValue())
+    financing_id = db.Column('financing_id', db.Integer,
+                             db.Sequence('financing_id_seq'), primary_key=True)
+    state_type_cd = db.Column('state_type_cd', db.String(3), nullable=False)
+                              #, db.ForeignKey('state_type.state_type_cd'))
+    registration_num = db.Column('registration_number', db.String(10), nullable=False)
+    life = db.Column('life', db.Integer, nullable=True)
+    expire_date = db.Column('expire_date', db.Date, nullable=True)
```

Comment:
are these just date, or date times?

Reply:
Bob created the table definitions as all Dates.  Where we care about the timestamp the time is saved correctly by saving as SQLAlchemy datetime

Reply:
To clarify, expire_date as a Date is correctly saving date and time information in the Oracle database. Both Date and DateTime store time information with Oracle Date table column types. The spec defines expiry date as a Date which the model reflects. When developing the financing and renewal statements the model can be revised.

Reply:
we'll migrate these to DateTime next sprint, and as the DB is not currently being created from the model, this should not be an issue.